src/sage/modular/hypergeometric_motive.py

```diff
@@ -1131,6 +1137,44 @@ def canonical_scheme(self, t=None):
         ideal = ring.ideal([eq0, eq1, self.M_value() * eq2_neg - t * eq2_pos])
         return Spec(ring.quotient(ideal))
 
+    def polytope(self):
```

Comment:
I would suggest to call it `lattice_polytope`
and provide a method `polyhedron` that returns a Polyhedron

Reply:
I prefer to have just one method.

Reply:
Makes sense, I don't question your motives ;)